README.md

```diff
@@ -41,15 +41,16 @@ make fmt.write
 
 ## Supported Languages
 
-| Language   | Author Credit                                 |
-| ---------- | --------------------------------------------- |
-| English    | [bndw](https://github.com/bndw)               |
-| Chinese    | [Baoyuantop](https://github.com/Baoyuantop)   |
-| Spanish    | [oscfdezdz](https://github.com/oscfdezdz)     |
-| Portuguese | [pedrorenan](https://github.com/pedrorenan)   |
-| Japanese   | [hatsu38](https://github.com/hatsu38)         |
-| Russian    | [Teraskull](https://github.com/Teraskull)     |
-| Ukrainian  | [Teraskull](https://github.com/Teraskull)     |
-| Dutch      | [wouterbrink](https://github.com/wouterbrink) |
-| French     | [Divlo](https://github.com/Divlo)             |
-| Turkish    | [Riza Ergun](https://github.com/rizaergun)    |
+| Language   | Author Credit                                      |
+| ---------- | ---------------------------------------------------|
+| English    | [bndw](https://github.com/bndw)                    |
+| Chinese    | [Baoyuantop](https://github.com/Baoyuantop)        |
+| Spanish    | [oscfdezdz](https://github.com/oscfdezdz)          |
+| Portuguese | [pedrorenan](https://github.com/pedrorenan)        |
+| Japanese   | [hatsu38](https://github.com/hatsu38)              |
+| Russian    | [Teraskull](https://github.com/Teraskull)          |
+| Ukrainian  | [Teraskull](https://github.com/Teraskull)          |
+| Dutch      | [wouterbrink](https://github.com/wouterbrink)      |
+| French     | [Divlo](https://github.com/Divlo)                  |
+| Turkish    | [Riza Ergun](https://github.com/rizaergun)         |
+| German     | [Youssef Mahmoud](https://github.com/YoussefMahmod)|
```

Comment:
In my opinion it should mention both @YoussefMahmod  and @devofthings as they both made nontrivial additions to the german translations.

```suggestion
| German     | [Youssef Mahmoud](https://github.com/YoussefMahmod)<br>[devofthings](https://github.com/devofthings) |
```
This suggestions is not enough of course as i can only edit one of the lines and that would mess up the formatting of the table

Reply:
I apologize for these mistakes, I am new to open sourcing and this was my second PR ever, so thanks for correct me and i hope you to accept my apologize.